content/sensu-go/6.2/operations/control-access/create-limited-service-accounts.md

```diff
@@ -0,0 +1,332 @@
+    "created_by": "admin",
+    "name": "ec2-service-delete",
+    "namespace": "default"
+  },
+  "spec": {
+    "role_ref": {
+      "name": "ec2-delete",
+      "type": "Role"
+    },
+    "subjects": [
+      {
+        "name": "ec2-service",
+        "type": "User"
+      }
+    ]
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+4. Create an API key for the `ec2-service` user:
+
+   {{< code shell >}}
+sensuctl api-key grant ec2-service
+{{< /code >}}
+
+   The response will include an API key that is assigned to the `ec2-service` user, which you will need to configure the EC2 handler.
+
+The `ec2-service` limited service account is now ready to use with the [Sensu EC2 Handler][3] integration.
+
+## Add the EC2 handler dynamic runtime asset
+
+To power the handler to remove AWS EC2 instances, use sensuctl to add the [Sensu Go EC2 Handler][13] [dynamic runtime asset][14]:
+
+{{< code shell >}}
+sensuctl asset add sensu/sensu-ec2-handler:0.4.0
+{{< /code >}}
+
+The response will indicate that the asset was added:
+
+{{< code shell >}}
+fetching bonsai asset: sensu/sensu-ec2-handler:0.4.0
+added asset: sensu/sensu-ec2-handler:0.4.0
+
+You have successfully added the Sensu asset resource, but the asset will not get downloaded until
+it's invoked by another Sensu resource (ex. check). To add this runtime asset to the appropriate
+resource, populate the "runtime_assets" field with ["sensu/sensu-ec2-handler"].
+{{< /code >}}
+
+You can also download the dynamic runtime asset definition for Debian or Alpine from [Bonsai][13] and register the asset with `sensuctl create --file filename.yml`.
+
+{{% notice note %}}
+**NOTE**: Sensu does not download and install dynamic runtime asset builds onto the system until they are needed for command execution.
+Read [the asset reference](../../../plugins/assets#dynamic-runtime-asset-builds) for more information about dynamic runtime asset builds.
+{{% /notice %}}
+
+## Configure an EC2 handler for the service account
+
+To configure the EC2 handler, you will need AWS account credentials and details for the AWS instance you want to manage, like the AWS instance ID.
+You will also need the API key for the `ec2-service` user.
+
+{{% notice note %}}
+**NOTE**: Use [secrets management](../../manage-secrets/secrets-management/) to configure environment variables for your AWS access and secret keys and the `ec2-service` user's API key and avoid exposing this sensitive information by listing it directly in the EC2 handler definition.<br><br>
+The [Sensu Go EC2 Handler's Bonsai page](https://bonsai.sensu.io/assets/sensu/sensu-ec2-handler#environment-variables) includes an example for configuring secrets definitions with Sensu's built-in [`env` secrets provider](../../manage-secrets/secrets-providers/#env-secrets-provider-example).
+{{% /notice %}}
+
+In the following code, replace the placeholders for these attributes with valid values:
+
+- `aws-region`: the AWS region where your EC2 instance is located.
```

Comment:
Can you specify more than one region in a comma-separated list or something like that?

Reply:
No, you cannot.